README.rst

```diff
@@ -80,3 +83,54 @@ a-json-file.json
 
 ```NewUsers`` must be added to a list of components executed in :func:`djangocms_fil_bootstrap.bootstrap`.
 
+
+==============
+Data structure
+==============
+Note that it is possible to conform to DRY principles *to a degree*. If you wish to accumulate data, then records in later files will still need to be defined but they do not require all the data to be present, providing that your component uses `get_or_create` when generating new records.
+
+E.g. Let's say you have two json files to import: `roles.json` and `demo.json`. You may define all the permissions and roles in the first file. Any records which may be required in the second just need their unique identifier property, so that `get_or_create` will execute a SELECT instead of an INSERT.
+
+Note that each root key in your json data file must correspond by name with one of the components.
```

Comment:
Actually, it needs to correspond to Component.field_name attribute set in your component class. Also, you can technically read from other keys as well by using `self.bootstrap.data("other_key")`

Reply:
Isn't it `self.bootstrap.data["other_key"]`? (square brackets)... at least that's what we decided was best in collections.py